include/mbedtls/threading.h

```diff
@@ -47,6 +47,19 @@ typedef struct mbedtls_threading_mutex_t {
 } mbedtls_threading_mutex_t;
 #endif
 
+#if defined(MBEDTLS_THREADING_SRWLOCK)
+/*
+ * Use spaces to pass the macroses name check.
+ */
+#   define WIN32_LEAN_AND_MEAN
```

Comment:
Please add `WIN32_LEAN_AND_MEAN` and `_WIN32_WINNT` to the `exclusions` list in `parse_macros` in `tests/scripts/check_names.py`. That's how we deal with the similar case of `_CRT_SECURE_NO_DEPRECATE`.